### ISSUE TITLE
rcmgr: flaky TestResourceManagerConnInbound

### ISSUE THREAD

**Issue by marten-seemann** [libp2p/go-libp2p#1671]
```
  === RUN   TestResourceManagerConnInbound
      rcmgr_test.go:79: expected ResourceManager to block incoming connection
  --- FAIL: TestResourceManagerConnInbound (3.76s)

**Comment by marten-seemann** [comment]
Apparently this only applies to this branch: https://github.com/libp2p/go-libp2p/pull/1615

**Comment by BigLep** [comment]
This will get addressed as part of landing #1615.
